docs/API.md

```diff
@@ -32,6 +24,16 @@ module.exports = async function() {
 }
 ```
 
+Then you need to set up the player. This usually takes less than a second:
+```typescript
+import TrackPlayer from 'react-native-track-player';
+
+await TrackPlayer.setupPlayer()
+// The player is ready to be used
+```
+
+Make sure the setup method has completed before interacting with any other methods on `TrackPlayer` in order to avoid instability.
```

Comment:
```suggestion
Make sure the setup method has completed before interacting with any other functions in `TrackPlayer` in order to avoid instability.
```